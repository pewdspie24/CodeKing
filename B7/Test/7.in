15
28 905738870
8352063 8626795
7776904 3980665
8599916 3554190
3690036 6778674
5587766 9785675
6847267 6030431
7993999 8356191
8043025 6422165
1293928 1289424
9452851 446962
9545852 4896715
3989945 9848981
3621259 6470856
8062352 391852
4217856 2058907
3650329 1882649
9689726 9308457
6817072 1435648
6014508 7104333
8092681 1357658
7600285 3646765
6501906 9905601
345599 7404541
2319701 1608190
4138137 6425369
6835350 3759505
3538245 9991531
9001101 6062554
20 562347896
3022563 3260626
1187160 2923738
3586306 5415095
9559784 7814487
791912 4553771
6439976 9361169
6797703 1064624
4899644 1038950
3611196 2210224
5697243 3416497
2129764 4702243
7693461 5325886
9919874 3244869
6365130 693794
2006659 9319324
9840994 7322911
4009180 460192
5366533 6492502
1261316 4024284
136245 9134042
21 71189635
3756235 3362039
6452157 4747720
7647196 6829274
4159918 7226897
8859065 947156
5048437 5438485
4410383 6665381
3241581 7461408
249755 467644
2312115 9538250
2390910 6910237
6201647 7244094
6669986 7561071
9704658 2356502
7188858 3981584
8750517 3093895
3725127 1009277
3966876 5007984
5356843 299921
3271787 155738
9228325 1452344
13 647451514
1138193 8005409
9215600 1636657
5459420 9948412
5574542 2534186
2064021 9204704
3246587 2378911
1866021 2280946
7067996 8052729
6560855 785784
6564242 7860277
9432744 4346793
7825831 3625363
4546457 219548
20 742202449
8218193 5407774
4810687 6811458
8504469 8720274
1315299 9188391
3382824 5479136
9562940 4573417
9896439 9463045
4515596 4387728
7388503 1561362
1867974 6320364
9295691 5671373
6944812 7205775
8443197 3281271
9956274 9370121
5337940 2697924
309963 7656729
1348769 5885212
8195664 2929557
588119 217232
1103066 5644779
23 904920814
7694582 1549825
6928947 221728
1144794 1271249
5052517 1221332
1835908 8682090
8382823 7894953
2449242 950139
8552299 3124153
1792829 6726109
2630726 7902215
6130357 8821246
3619562 6421807
3111477 4101638
4620719 9620402
8694446 8639182
7121681 7542861
7465349 1012229
8406359 5416612
9000789 9052466
5330892 1573117
3158038 8613501
6181818 8688853
5674476 2569094
25 578021268
4590357 1347872
8349516 3993938
2915100 5972033
2511364 9121604
9378907 934742
8036426 4885075
2218661 9715164
4890300 9899904
6382686 6550073
1900383 9488327
4959035 9328515
6047589 1790152
8066360 7216600
9043742 262735
7285242 3561977
7044493 3930460
6509310 5959619
4512707 5728712
4269665 4097890
5193334 5478396
3283716 1586347
4755584 9539404
7090544 343445
4815387 4703513
1038704 2743337
11 210512630
1115037 8328929
6274826 1222109
3648589 3956042
5391037 4832444
7960977 6645479
3351858 1559450
1137841 6977283
5500724 8499059
8968235 2492541
3837821 2001196
3691459 9229476
17 761733469
3771559 2719194
8394676 6572636
9736346 9702596
4722600 1160255
6367700 8061805
7356567 9172207
1625962 9626084
6637505 246140
5160429 5690197
6999290 9958151
3838122 3773161
8001433 8755961
5250178 1748230
111473 8946509
7656163 318934
6612208 1604231
8155928 658752
21 461833148
3079489 1281486
6522461 575654
1300905 9320576
9273258 6772212
700109 1183474
2831418 4182016
7276208 8700749
3265759 1803552
1615723 9357537
6423958 2391748
4744913 588021
3986041 4391979
4668076 240364
1225944 9590821
2255082 4889493
2431421 8792920
2535093 6801537
4902533 346848
6010108 3574926
6077975 1511632
7716630 7411266
26 76038733
9575369 8375539
4835090 8970235
4768361 1832405
5585794 7048319
5282702 9737989
6201812 1945231
4646591 8063231
4071644 2102831
1987540 8359042
3640048 491751
2623358 9625381
2638912 2897353
9644495 5827531
3175415 4596273
7804978 2249222
9481183 471492
3134885 5767901
456619 5440543
1269974 3295918
5340425 4060658
3327751 9917054
6128864 885455
2371363 7285405
9013311 3464453
6892438 1566308
219440 5019002
20 67228399
5241199 978587
8077474 1024229
7992241 6480262
8901166 591972
4955786 1526484
5328698 904123
5522580 5042685
1302395 4274694
676486 3004899
6061036 6916766
3400581 2424734
8289121 1706292
3518622 7556172
4905423 809032
4491320 3405608
8762074 8548743
5473820 7231885
4734318 1544223
4795197 5029215
5959020 4996969
23 262267433
8863340 550789
1611180 6322649
4082254 9779430
2586641 2659295
7069812 962000
4737274 5380549
9811609 8185447
7124345 1949262
8590675 3428938
7279554 6567929
1509256 5864631
3379283 2771070
3097030 712162
7007490 2091329
8932262 1161980
3030011 4838961
248216 1050128
1472602 8166449
8314850 9616091
9456546 1022101
8118185 4888136
1755077 5869465
9602097 6801308
12 355144356
1227162 9563454
8924428 5912070
1914038 9576430
6872045 9388740
3523029 4780533
453958 124459
115599 1110180
8235571 5572655
8449215 6954792
8364882 9938166
2513343 5922928
4555443 5833920
14 785095054
590656 6927591
4120241 9206966
4402503 6594316
5828534 3653024
2070365 4615472
1335385 1886846
5643700 8814686
8265708 5046680
2091187 1569410
4176264 9060129
8072685 6118414
6226792 7270731
2857269 5800494
7973699 1113703
