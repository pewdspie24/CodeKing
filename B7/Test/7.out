101
84
9
86
127
135
66
33
102
78
9
9
31
116
115
